parent-pom/pom.xml

```diff
@@ -324,6 +324,11 @@
                         <groupId>org.xerial.snappy</groupId>
                         <artifactId>snappy-java</artifactId>
                     </exclusion>
+                    <!-- Excluded to fix CVE-2024-25638 -->
+                    <exclusion>
+                        <groupId>dnsjava</groupId>
+                        <artifactId>dnsjava</artifactId>
+                    </exclusion>
```

Comment:
Why do you exclude the dependency? It should be enough to just update it?

Reply:
You're right, exclusion is not needed, removed